Services/Dashboard/Block/js/replaceModalContent.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @param {string} id
+ * @param {int} view
+ * @param {string} url
+ */
+const replaceModalContent = (function (id, view, url) {
+    document.getElementById(id).addEventListener('click', function (e) {
+        const form = document.querySelector('form[name="pd_remove_multiple_view_' + view + '"]');
+        const formData = new FormData(form);
+        let selected_ids = [];
+        for (const [name, value] of formData) {
+            selected_ids.push(value);
+        }
+        const modal = document.querySelector('div[data-modal-name="remove_modal_view_' + view + '"]');
+        let post_data = '';
+        for (let i = 0; i < selected_ids.length; i++) {
```

Comment:
This for loop and the `selected_ids` variable are unnecessary. You can just create the post data in the previous loop in line `12`.